configuration/src/main/java/com/synopsys/integration/configuration/config/PropertyConfiguration.java

```diff
@@ -60,6 +63,31 @@ public PropertyConfiguration(@NotNull final List<PropertySource> orderedProperty
         this.orderedPropertySources = orderedPropertySources;
     }
 
+    public List<PropertyInfo> collectPropertyInfo(List<Property> knownProperties, boolean maskPasswords) {
+        List<Property> sortedProperties = sortProperties(knownProperties);
```

Comment:
I would argue this shouldn't sort the properties - the consumer of the list can sort it.